### PR TITLE
@skylar/price impact followup

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes APP-1109

## What changed (plus any additional context for devs)
context in the ticket. most of the code is adapted from the BX

did a lot more QA this time 
## Screen recordings / screenshots
wrap eth: 
eth to erc20: 

erc20 to eth: 

erc20 bridge: 

eth bridge: 

culprit eth bridge:

### CODE REVIEW

File: src/handlers/swap.ts [hunk 1]
```diff
@@ -240,6 +240,47 @@ export const getSwapGasLimitWithFakeApproval = async (
...
+  console.log({
+    isUnwrap:
+      sellTokenAddress.toLowerCase() === WRAPPED_ASSET[chainId].toLowerCase() &&
+      buyTokenAddress.toLowerCase() === ETH_ADDRESS_AGGREGATORS.toLowerCase(),
+  });
+  return (
+    sellTokenAddress.toLowerCase() === WRAPPED_ASSET[chainId].toLowerCase() &&
+    buyTokenAddress.toLowerCase() === ETH_ADDRESS_AGGREGATORS.toLowerCase()
+  );
+};
```
Comment (walmat) [inline src/handlers/swap.ts:261]:
```suggestion
```

File: src/handlers/swap.ts [hunk 2]
```diff
@@ -240,6 +240,47 @@ export const getSwapGasLimitWithFakeApproval = async (
...
+    isWrap:
+      sellTokenAddress.toLowerCase() ===
+        ETH_ADDRESS_AGGREGATORS.toLowerCase() &&
+      buyTokenAddress.toLowerCase() === WRAPPED_ASSET[chainId].toLowerCase(),
+  });
+  return (
+    sellTokenAddress.toLowerCase() === ETH_ADDRESS_AGGREGATORS.toLowerCase() &&
+    buyTokenAddress.toLowerCase() === WRAPPED_ASSET[chainId].toLowerCase()
+  );
+};
```
Comment (walmat) [inline src/handlers/swap.ts:282]:
```suggestion
```

**Review by walmat — APPROVED** [review]
cleanup but lgtm

File: src/handlers/swap.ts (hunk 2 above)
Comment (skylarbarrera) [reply src/handlers/swap.ts:282]: ty

**Pull request merged**